app/components/ontology_browse_card_component.rb

```diff
@@ -17,7 +17,7 @@ def ontology
   end
 
   def external_ontology?
-    !internal_ontology?(@ontology[:id]) || (Array(@ontology[:sources]).size > 1)
```

Comment:
why removing this `|| (Array(@ontology[:sources]).size > 1)` ?

Reply:
The ontologies that are present in the current portal, we count them internal. 
I noticed that this change break one thing, which is the portals buttons when the ontology is internal and also present in other portals and I fixed it in the commit (https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/778/commits/7db6a96f1bad0f3209ccc794174f1fd904c344d7)

Reply:
okay